main.go

```diff
@@ -48,16 +49,24 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var eventsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	var ociRegistryAddr string
+	var (
+		metricsAddr                    string
+		eventsAddr                     string
+		enableLeaderElection           bool
+		probeAddr                      string
+		ociRegistryAddr                string
+		ociRegistryCertificateLocation string
+		ociRegistryInsecureSkipVerify  bool
+		ociRegistryNamespace           string
+	)
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&eventsAddr, "events-addr", "", "The address of the events receiver.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&ociRegistryAddr, "oci-registry-addr", ":5000", "The address of the OCI registry.")
+	flag.StringVar(&ociRegistryCertificateLocation, "oci-registry-certificate-location", v1alpha1.DefaultRegistryCertificateLocation, "The location in the contain where certificates live for the registry.")
```

Comment:
can we use those instead:
```
       --ca-file string             certificate authority file path
       --cert-file string           certificate file path
       --key-file string            key file path
```

People are use to seeing those, and it's easier to understand what's requested.